core/libsilky/silky/columnw.cpp

```diff
@@ -17,6 +17,18 @@ ColumnW::ColumnW(DataSetW *parent, MemoryMapW *mm, ColumnStruct *rel)
     _mm = mm;
 }
 
+void ColumnW::setName(const char *name)
+{
+    ColumnStruct *s = struc();
+    int length = strlen(name)+1;
+    char *chars = _mm->allocate<char>(length);
+
+    std::memcpy(chars, name, length);
+
+    s->name = _mm->base(chars);
```

Comment:
`s` was assigned before an allocate, so it needs to be resolved again before it can be used